delete 버튼 눌러도 동작 안함.

로그 확인해보니 아래와 같은 에러 발생하고 있음.

```
2022-05-25T07:32:30.246Z	625b722f-3009-4347-a083-d91b99a0e18b	ERROR	Error: An API error occurred: invalid_arguments
    at Object.t.platformErrorFromResult (/var/task/src/handler-slack.js:2:86646)
    at S.apiCall (/var/task/src/handler-slack.js:2:82194)
    at processTicksAndRejections (internal/process/task_queues.js:95:5)
    at async t.openViewToDelete (/var/task/src/handler-slack.js:2:741936)
    at async l (/var/task/src/handler-slack.js:2:680600)
    at async Array.<anonymous> (/var/task/src/handler-slack.js:2:23128)
    at async Array.t.onlyActions (/var/task/src/handler-slack.js:2:21916) {
  code: 'slack_webapi_platform_error',
  data: {
    ok: false,
    error: 'invalid_arguments',
    response_metadata: { messages: [Array], scopes: [Array] }
  }
```

```
2022-05-25T07:32:28.654Z	bfd2f106-a0d1-4890-9c32-1515e3abee8e	ERROR	
{
    "code": "slack_webapi_platform_error",
    "data": {
        "ok": false,
        "error": "invalid_arguments",
        "response_metadata": {
            "messages": [
                "[ERROR] failed to match all allowed schemas [json-pointer:/view]",
                "[ERROR] failed to match all allowed schemas [json-pointer:/view/title]",
                "[ERROR] must be less than 25 characters [json-pointer:/view/title/text]"
            ],
            "scopes": [
                "chat:write.customize",
                "chat:write",
                "groups:write",
                "commands",
                "app_mentions:read",
                "groups:read",
                "channels:read",
                "im:write",
                "mpim:read",
                "im:read"
            ]
        }
    }
}
```

--- a/packages/slack-app/src/handler-socket-mode.ts
+++ b/packages/slack-app/src/handler-socket-mode.ts
@@ -13,5 +13,5 @@ const boltApp = new App({
   socketMode: true,
 });
 
-boltApp.start().then(() => console.log('⚡️ Bolt app started'))
+boltApp.start().then(() => console.log('⚡️ Socket mode Bolt app started'))
 initialize(boltApp)

--- a/packages/universal/package.json
+++ b/packages/universal/package.json
@@ -5,7 +5,7 @@
   "scripts": {
     "build": "tsc",
     "clean": "rimraf dist",
-    "dev": "tsc --watch",
+    "dev": "tsc --watch --preserveWatchOutput",
     "create-dynamodb-tables": "ts-node bin/create-dynamodb-table.ts"
   },
   "main": "dist/index.js",

--- a/packages/universal/src/core/core-common.ts
+++ b/packages/universal/src/core/core-common.ts
@@ -242,8 +242,6 @@ export const openViewToDelete = async (web: WebClient, payload: IMyBlockActionPa
     channelId: channel.id, ts, threadTs: thread_ts,
     channelName: channel.name, responseUrl: response_url,
   }
-  const group: IGroup = await getGroup(channel.id, { cache: true })
-  const m = getMessage(group.lca2)
   await web.views.open(getDeletionViewOpenArg(trigger_id, pm))
 }
 

--- a/packages/universal/src/messages/en.ts
+++ b/packages/universal/src/messages/en.ts
@@ -64,8 +64,8 @@ export const messages: Messages = {
   STR_REPORT: ':rotating_light:Report',
   STR_REPORT_N: ':rotating_light:Report %d',
 
-  STR_VIEW_TITLE_VOICE_DELETION: 'Delete anonymous messages',
-  STR_VIEW_TITLE_REPLY_DELETION: 'Delete anonymous comments',
+  STR_VIEW_TITLE_VOICE_DELETION: 'Delete a message',
+  STR_VIEW_TITLE_REPLY_DELETION: 'Delete a comment',
   STR_VIEW_DELETE: 'Delete',
   STR_VIEW_CANCEL: 'Cancel',
   STR_NOT_MATCHED_PASSWORD: 'Passwords do not match.',

